include/database/MysqlManager.php

```diff
@@ -1506,13 +1548,19 @@ public function createDbUser($database_name, $host_name, $user, $password)
      */
     public function createDatabase($dbname)
     {
-        $this->query("CREATE DATABASE `$dbname` CHARACTER SET utf8 COLLATE utf8_general_ci", true);
+        $collation = $this->getCollation();
+        $charset = $this->getCharset();
+
+        $this->query("CREATE DATABASE `$dbname` CHARACTER SET $charset COLLATE $collation", true);
```

Comment:
Please use quote or quoted methods on variable before concatenate it into an sql query.